src/base/pegasus_const.h

```diff
@@ -62,4 +62,6 @@ extern const std::string PEGASUS_CLUSTER_SECTION_NAME;
 extern const std::string ROCKSDB_ENV_SLOW_QUERY_THRESHOLD;
 
 extern const std::string ROCKSDB_ITERATION_THRESHOLD_TIME_MS;
+
+extern const std::string SPLIT_VALIDATE_PARTITION_HASH;
```

Comment:
```suggestion
extern const std::string SPLIT_CHECK_PARTITION_HASH;
```

Reply:
Actually, the app envs is called `replica.split.validate_partition_hash`, so I rename all `check_partition_hash` into `validate_partition_hash`